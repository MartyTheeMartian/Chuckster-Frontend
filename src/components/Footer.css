footer {
  position: fixed;
  bottom: 0;
  width: 100%;
  padding: 4px 0;
  text-align: center;
  font-size: 10px;
}
