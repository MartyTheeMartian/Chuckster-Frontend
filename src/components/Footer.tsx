import './Footer.css';

export default function Footer(): JSX.Element {
  return (
    <footer>
      Â© 2022 Chuckster
    </footer>
  );
}
