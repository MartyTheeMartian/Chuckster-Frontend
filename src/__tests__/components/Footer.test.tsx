import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Footer from 'components/Footer';

it('renders App component', () => {
  const result = shallow(<Footer />);
  expect(toJson(result)).toMatchSnapshot();
});
