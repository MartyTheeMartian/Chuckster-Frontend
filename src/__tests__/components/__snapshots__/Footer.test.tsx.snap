// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders App component 1`] = `
<footer>
  Â© 2022 Chuckster
</footer>
`;
